tests/longevity-tests/Dockerfile.foundation

```diff
@@ -18,12 +18,12 @@ COPY secrets /go/src/github.com/vmware/vic
 ENV LONGEVITY=1
 ENV GOVC_INSECURE=1
 ENV GOVC_USERNAME=administrator@vsphere.local
-ENV GOVC_DATASTORE=vsanDatastore
-ENV TEST_DATASTORE=vsanDatastore
+ENV GOVC_DATASTORE=CPBU_2TB_01
+ENV TEST_DATASTORE= CPBU_2TB_01
 ENV TEST_USERNAME=administrator@vsphere.local
 ENV TEST_TIMEOUT=3m
-ENV TEST_RESOURCE=cls
+ENV TEST_RESOURCE=Cluster
 ENV DOMAIN=
-ENV BRIDGE_NETWORK=bridge
-ENV PUBLIC_NETWORK=vm-network
+ENV BRIDGE_NETWORK=vxw-dvs-3444-virtualwire-1-sid-5000-VCH-LS-1
+ENV PUBLIC_NETWORK=DPortGroup
 ENV DOCKER_API_VERSION=1.23
```

Comment:
Also, can we consider getting log publication to GCP added to Longevity. That could be in a separate PR.

Final note, no need to run [Full CI] on this, this code is not executed at all in the CI. It could save some time :)

Reply:
yea, different PR as you said.  I will get it done with the DOCKER_API update